src/coreclr/src/vm/arm64/asmhelpers.S

```diff
@@ -473,6 +476,35 @@ LOCAL_LABEL(Exit):
     ret  lr
 WRITE_BARRIER_END JIT_WriteBarrier
 
+    // Begin patchable literal pool
+    .balign 64  // Align to power of two at least as big as patchable literal pool so that it fits optimally in cache line
+WRITE_BARRIER_ENTRY JIT_WriteBarrier_Table
+LOCAL_LABEL(wbs_begin):
+LOCAL_LABEL(wbs_card_table):
+    .quad 0
```

Comment:
That is very true.  movz/movn/movk can be optimized to occur early, so they often effectively take no time and no resources (except instruction cache).

Reply:
The disadvantage of changing code (rather than date) was that all the threads needed to be stopped.  The data approach allowed these updates to happen more asynchronously.

I suspect the code approach might still be better than data based on the percent of arm64 profile time in WriteBarrier et al.  But it was still worth mentioning.